__tests__/ncgamesapp.test.js

```diff
@@ -67,3 +67,77 @@ describe("GET:/api/reviews/:review_id", () => {
       });
   });
 });
+describe("PATCH /api/reviews/:review_id", () => {
+  it("accepts requests with votes update, returns an updated review", () => {
+    const newVote = { inc_votes: 1 };
+    const expectedReview = {
+      review_id: 1,
+      title: "Agricola",
+      category: "euro game",
+      designer: "Uwe Rosenberg",
+      owner: "mallionaire",
+      review_body: "Farmyard fun!",
+      review_img_url:
+        "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+      created_at: "2021-01-18T10:00:20.514Z",
+      votes: 2,
+    };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(newVote)
+      .expect(202)
```

Comment:
a `202` means the request has been accepted but doesn't mean that the server has done anything about it!

a `200` might be better here

[read more](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)

Reply:
changed!